config/locales/en.yml

```diff
@@ -1259,6 +1293,20 @@ en:
       rest: 'REST API'
   formtastic:
     hints:
+      numbering_period: It will only change the friendly ID in invoices, not the billing period.
+      charging_enabled:
+        Enables credit card transactions. When this setting is on, all due invoices will be
+        charged using the selected payment gateway. If you leave this setting off, billing will
+        take place and invoices will be issued, but no real transaction will take place.
+      currency:
+        Please make sure that the selected currency is supported by your payment gateway. If you are
+        missing currency options, don't hesitate to contact us.
```

Comment:
```suggestion
      finance/billing_strategy:
        numbering_period: It will only change the friendly ID in invoices, not the billing period.
        charging_enabled:
          Enables credit card transactions. When this setting is on, all due invoices will be
          charged using the selected payment gateway. If you leave this setting off, billing will
          take place and invoices will be issued, but no real transaction will take place.
        currency:
          Please make sure that the selected currency is supported by your payment gateway. If you are
          missing currency options, don't hesitate to contact us.
```

Reply:
Awesome